src/api/firebase.js

```diff
@@ -209,3 +209,58 @@ export async function deleteItem() {
 	 * this function must accept!
 	 */
 }
+
+export function comparePurchaseUrgency(array) {
+	const sorted = array.sort((a, b) => {
+		const dateA = Math.floor(
```

Comment:
May be cool to adda a few comments through this stage of the algorithm too.

Reply:
Do you mean leave comments through each step of the function?

Reply:
Thanks, this is a great idea. I just added explanatory comments for this function.